.drone.yml

```diff
@@ -215,30 +215,110 @@ steps:
           fi
         fi
 
-  - name: Run docs tests
+  - name: Run docs tests (internal links only)
     image: golang:1.14.4
     commands:
       - |
         cd /go/src/github.com/gravitational/teleport
         git diff --raw ${DRONE_COMMIT}..origin/${DRONE_COMMIT_BRANCH:-master} | awk '{print $6}' | grep -E '^docs' | grep -v ^$ | cut -d/ -f2 | sort | uniq > /tmp/docs-versions-changed.txt
         if [ $(stat --printf="%s" /tmp/docs-versions-changed.txt) -gt 0 ]; then
-          echo "Changes to docs detected, versions $(cat /tmp/docs-versions-changed.txt | tr '\n' ' ')"
+          echo "---> Changes to docs detected, versions $(cat /tmp/docs-versions-changed.txt | tr '\n' ' ')"
           # Check trailing whitespace
           make docs-test-whitespace
           # Check links
           for VERSION in $(cat /tmp/docs-versions-changed.txt); do
             if [ -f docs/$VERSION/milv.config.yaml ]; then
+              go get github.com/magicmatatjahu/milv
               cd docs/$VERSION
-              echo "Running milv on docs/$VERSION:"
-              go get -u github.com/magicmatatjahu/milv
-              milv
-              echo "------------------------------"
+              echo "---> Running milv on docs/$VERSION:"
+              milv -ignore-external
```

Comment:
In this case do we keep the local `milv.config.yaml` as `ignore-external: true` and this overrides it?

Reply:
Yes, exactly. Command-line switches to `milv` override config file defaults.